src/uwtools/tests/config/formats/test_fieldtable.py

```diff
@@ -49,3 +69,25 @@ def test_fieldtable_simple(config, ref, tmp_path):
     FieldTableConfig(config=config).dump(outfile)
     with open(outfile, "r", encoding="utf-8") as out:
         assert out.read().strip() == ref
+
+
+def test_fieldtable_as_dict():
+    d1 = {"section": {"key": "value"}}
+    config = FieldTableConfig(d1)
+    d2 = config.as_dict()
+    assert d2 == d1
+    assert isinstance(d2, dict)
+
+
+def test_fieldtable_dump(dumpkit):
+    d, expected, path = dumpkit
+    FieldTableConfig(d).dump(path)
+    with open(path, "r", encoding="utf-8") as f:
+        assert f.read().strip() == expected
+
+
+def test_fieldtable_dump_dict(dumpkit):
+    d, expected, path = dumpkit
+    FieldTableConfig.dump_dict(d, path=path)
+    with open(path, "r", encoding="utf-8") as f:
+        assert f.read().strip() == expected
```

Comment:
Here and below, I added unit tests for the new `as_dict()` method, as well as for the existing `dump()` and `dump_dict()` methods. The latter two were already transitively tested, but now have their own direct tests, too.